application/src/main/java/org/togetherjava/tjbot/features/Features.java

```diff
@@ -140,7 +141,9 @@ public static Collection<Feature> createFeatures(JDA jda, Database database, Con
         features.add(new HelpThreadCommand(config, helpSystemHelper));
         features.add(new ReportCommand(config));
         features.add(new BookmarksCommand(bookmarksSystem));
-        features.add(new ChatGPTCommand(config));
+
+        ChatGPTService chatGPTService = new ChatGPTService(config);
```

Comment:
don't make this object here, see the top of this method, move it at the end of those objects i.e. line 76